### DISCUSSION TITLE
Plans to submit PR to master?

### DISCUSSION THREAD

**Discussion by kylerw** [Desuuuu/klipper#69] [Q&A]
Would love to be able to pull this into KIAUH and standard install/upgrade methods.

**Comment by Desuuuu** [comment]
I don't plan on doing it for the current version. Once the new version is available, I'll look into it.

**Comment by kylerw** [comment]
Do you mean new Klipper version?

**Reply by Desuuuu** [reply]
I'm talking about a new interface/implementation for the touchscreen.

**Reply by Desuuuu** [reply]
The new version is now available for testing. If you're interested, you'll find the details about it [here](https://github.com/Desuuuu/klipper/discussions/74).
